vm/vmimpl/console.go

```diff
@@ -76,6 +76,33 @@ func (t *tty) Close() error {
 	return nil
 }
 
+func OpenRemoteKernelLog(ip string, console string) (rc io.ReadCloser, err error) {
+	rpipe, wpipe, err := osutil.LongPipe()
```

Comment:
Please add here a note that says that it's accessing not the Android VM itself, so oops messages should still pass through.